stock_picking_late_activity/README.rst

```diff
@@ -91,6 +91,7 @@ Authors
 ~~~~~~~
 
 * Tecnativa
+* Heliconia Solutions
```

Comment:
@HeliconiaSolutions You cannot put yourself as an author without major contribution to this module. Please put yourself in Contributors section (as individual).